src/components/sparks/Sparks.spec.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import {render} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import {testA11y} from '../../axe';
+import Sparks from './Sparks';
+
+// mock all animations WAAPI so tests run correctly in jsdom environment (no window)
+jest.mock('./animation', () => {
+  return {
+    useAnimation: jest.fn(() => ({
+      register: jest.fn(),
+      phase: 'initial',
+      setPhase: jest.fn(),
+    })),
+  };
+});
+
+describe('<Sparks />', () => {
+  it('renders correctly and displays its children', () => {
+    const sparks = render(
+      <Sparks>
+        <button type="button">Click me</button>
+      </Sparks>
+    );
+
+    expect(sparks.container.firstChild).toBeTruthy();
```

Comment:
If you add role on svg ('img') , you can use getByRole and check more accurately sparks.

Reply:
Yea, honestly the meaning of this test is vague (checks if element just renders anything)

I will remove it once I figure out chromatic tests

Reply:
As for role img it is pointless here since all contaniner is hidden in accesibility tree. The only thing I care about is that component renders. Expect is just used to satisfy test.

prpbably this test can be skipped because existence of 2 other tests, it was just the only test for some time.

Reply:
That's the best case if any other tests cover this we can get rid of this artificial "it renders" tests